CONTRIBUTING.md

```diff
@@ -156,10 +156,11 @@ callback is set. A typical data download callback will look like
 @app.callback(self.plugin_data_output,
               [self.plugin_data_requested])
```

Comment:
```suggestion
@app.callback(self.plugin_data_output,
              self.plugin_data_requested)
```
The recent Dash versions (`>=1.15`) now supports input not being a list, so we can use this opportunity to simplify the guide probably. We already require `>=1.16` in `webviz-config` due to CSP and usage of https://github.com/plotly/dash/pull/1371.